docs/usage/self-hosting.md

```diff
@@ -144,6 +144,38 @@ spec:
               emptyDir: {}
 ```
 
+#### CircleCI
+
+If you are using CircleCI, you can use the third-party [daniel-shuy/renovate](https://circleci.com/developer/orbs/orb/daniel-shuy/renovate) orb to run a self-hosted instance of Renovate on CircleCI.
+
+By default, the orb looks for the self-hosted configuration file in the project root, but can be customized using the `config_file_path` parameter.
+
+Secrets should be configured using environment variables (eg. `RENOVATE_TOKEN`, `GITHUB_COM_TOKEN`).
+
+Configure environment variables in CircleCI Project Settings (https://circleci.com/docs/2.0/env-vars/#setting-an-environment-variable-in-a-project).
+To share environment variables across projects, use CircleCI Contexts (https://circleci.com/docs/2.0/contexts/).
```

Comment:
I think these should probably be cleaner links, you can do this by following this pattern:

```markdown
[text that describes where the link points to](https link)
```

Reply:
Ironically, I wanted to do that for the Orb documentation, but couldn't because CircleCI's Orb documentation doesn't support markdown (only plaintext). I copied this from the Orb documentation, but forgot to do that, thanks!

Reply:
Thanks for implementing my suggestions! :+1: